_includes/master/reqs-sys.md

```diff
@@ -31,19 +31,24 @@ to connect to, we provide instructions in the [installation documentation](./ins
 
 ## Network requirements
 
-{{site.prodname}} requires the network to allow the following types of traffic.
-
-| Traffic | Protocol | Port |
-| ------- | -------- | ---- |
-| BGP     | TCP      | 179  |
-| IPIP\*  | 4        | n/a  |
-
-\* {% if include.orch == "OpenStack" %} If your compute hosts connect directly
-   and don't use IPIP, you don't need to allow IPIP traffic. {% else %} Our
-   manifests enable IPIP by default.  If you disable IPIP, you won't need to
-   allow IPIP traffic. {% endif %} Refer to [Configuring
-   IP-in-IP](../../usage/configuration/ip-in-ip) for more information.
-
-> **Tip**: On GCE, you can allow this traffic using firewall rules. In AWS, use
-> EC2 security group rules.
-{: .alert .alert-success}
+Ensure that your hosts and firewalls allow the following traffic.
+
+| Configuration                                                                         | Host                | Connection type | Port/protocol                                                 |
+|---------------------------------------------------------------------------------------|---------------------|-----------------|---------------------------------------------------------------|
+| {{site.prodname}} networking                                                          | All                 | Bidirectional   | TCP 179                                                       |
+| {{site.prodname}} networking in IP-in-IP mode (default mode)                          | All                 | Bidirectional   | IP-in-IP, often represented by its protocol number `4`        |
+{%- if include.orch == "Kubernetes" %}
+| flannel networking                                                                    | All                 | Bidirectional   | UDP 4789 (default)                                            |
+| etcd datastore                                                                        | etcd hosts          | Incoming        | Varies per etcd configuration                                 |
```

Comment:
We should also call out the default of 2379, since that is what most people will use.

Reply:
Also, the etcd port is only needed if you're using the etcd datastore (not kdd, for example)

Reply:
The fact that they only need this if they are using the etcd datastore is supposed to be conveyed via the Configuration column

Reply:
Right, but I don't think that it is. It just says 'etcd datastore' not "If you're using the etcd datastore, open this port, otherwise this isn't needed".

For example, I read it and thought that when using Calico you always need to open the etcd port, which is wrong.

Reply:
Maybe we need to rename the column from "Configuration" which is a bit vague, to "When this is needed" and give more than one or two word entries.